docs/cdn.md

```diff
@@ -1596,10 +1627,36 @@ Response JSON:
 
 ## Dealing with pixel ratios
 
-When dealing with mulitiple device pixel ratios, you can 'multiply' the outputted size of the image by adding the variable name `?devicePixelRatio=[0-9]`.
+When dealing with mulitiple device pixel ratios, you can 'multiply' the outputted size of the image by adding the parameter `devicePixelRatio` to your query. `devicePixelRatio` can be anywhere from 1 to 3. Supplying this parameter will inform the CDN to multiply the output dimensions by that factor.
```

Comment:
Not your typo, but should be:

"When dealing with multiple device pixel ratios, you can 'multiply' the output size of the image by adding the..."